### PR TITLE
Specify cassandra-stress VMs via configs.

### LINKED ISSUE #490: Cassandra stress advertises that it doesn't require disks
But it does:

https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py#L97-L108

See #475.

### PR DESCRIPTION
**Pull request by cmccoy**
Fixes #490.
Partial work toward #512.

### CODE REVIEW

File: perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py [hunk 1]
```diff
@@ -55,18 +54,20 @@
 cassandra_stress:
   description: Benchmark Cassandra using cassandra-stress
   vm_groups:
-    default:
+    cassandra:
+      vm_spec: *default_single_core
+      disk_spec: *default_500_gb
```
Comment (voellm) [inline perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py]: Why has the size been hard coded for hte default data disk?  Why not call this "*default_data_disk"?

File: perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py [hunk 2]
```diff
@@ -55,18 +54,20 @@
 cassandra_stress:
   description: Benchmark Cassandra using cassandra-stress
   vm_groups:
-    default:
+    cassandra:
```
Comment (voellm) [inline perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py]: cassandra_cluster?

File: perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py (hunk 1 above)
Comment (cmccoy) [inline perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py]: This was discussed in #510; the decision there was to leave the current default in place and be explicit about the characteristics of the disks involved. @noahl - could you open an issue for your proposed changes to this?

File: perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py (hunk 2 above)
Comment (cmccoy) [inline perfkitbenchmarker/benchmarks/cassandra_stress_benchmark.py]: How about `cassandra_nodes`? That matches `hbase_ycsb` fairly well.

**Comment by cmccoy** [comment]
Note that this changes the metadata reported slightly - there will be separate entries for the client and server node groups.

**Comment by ehankland** [comment]
LGTM

**Pull request merged**